lib/Conversation/CommentForm/styles.scss

```diff
@@ -2,6 +2,7 @@
    Config
    ========================================================================== */
 $comment-form-padding: $layout-spacing-base/2;
+$comment-form-input-background-color: #FDFDFD !default;
```

Comment:
We should using colour settings instead of hardcoding values, can this use one of our settings?